### DISCUSSION TITLE
Why does Netmaker server (docker) use a custom script to configure and manage SSL certificates instead of using the Caddy's auto certificate management feature?

### DISCUSSION THREAD

**Discussion by luzfcb** [gravitl/netmaker#2541] [Q&A]
Hello, I intend to use the same Caddy instance that Netmaker uses, and I would like to understand better how Netmaker makes use of certificates and how and why.

https://github.com/gravitl/netmaker/blob/master/docker/Caddyfile#L3

**Comment by mattkasun** [comment]
To work around rate limits of certificate providers
